app/players/elves/elf_ranger.py

```diff
@@ -0,0 +1,15 @@
+from app.players.elves.elf import Elf
+
+
+class ElfRanger(Elf):
+    def __init__(self, nickname: str, musical_instrument: str,
+                 bow_level: int) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._bow_level = bow_level
+
+    def player_info(self) -> str:
+        return (f"elves ranger {self.nickname}."
```

Comment:
Returned text is different from the text in task
```suggestion
        return (f"Elf ranger {self.nickname}."
```